.flake8-doc

```diff
@@ -1,5 +1,6 @@
 [flake8]
-select=D,I
+ignore=D301
+select=D
```

Comment:
I wonder what these new options mean?

Reply:
D are all options that are about doc from the flake8-docstrings package.

D301 tells you to add r in front of your docstring if you have backslashes in it. In your case we do want to use it in a normal (not raw) string so I disabled it.

(I was just something from the WIP yesterday to detect fault params but pylint is better in that regard).